media/js/ccnmtljs/bb_control_views.js

```diff
@@ -119,6 +119,7 @@ var ManageInstructorsView = Backbone.View.extend({
     },
 
     addInstructor: function(evt) {
+        console.log("Inside addInstructor");
```

Comment:
console.logs can break the script on certain browsers if the debug window is not open.